applications/ecallmgr/src/ecallmgr_fs_authz.erl

```diff
@@ -83,6 +84,15 @@ maybe_authorize_channel(Props, Node) ->
             end
     end.
 
+-spec is_referred(wh_proplist()) -> 'true' | 'false'.
```

Comment:
@fenollp Incorrect, see http://erlang.org/doc/reference_manual/typespec.html

boolean() is defined as 'false' | 'true'